simapp/app.go

```diff
@@ -229,11 +232,12 @@ func NewSimApp(
 		&app.NFTKeeper,
 		&app.SlashingKeeper,
 		&msgServiceRouter,
+		&txBaseAppOption,
 	); err != nil {
 		panic(err)
 	}
 
-	app.App = appBuilder.Build(logger, db, traceStore, msgServiceRouter, baseAppOptions...)
+	app.App = appBuilder.Build(logger, db, traceStore, msgServiceRouter, append(baseAppOptions, txBaseAppOption)...)
```

Comment:
We really shouldn't need to be doing this.

Reply:
If I don't add this, it does not go through my break point in `module.go`. So, I am not certain why we shouldn't need it. Could you point me where this behavior should happen?

Reply:
> The type needs to be runtime.BaseAppOption

Ahhhh!!! I knew it was going to be something involving `IsManyPerContainerType` lol

Reply:
Sorry... did you guys try looking at the dependency graph?

Reply:
Thanks!